github/repos.go

```diff
@@ -1152,9 +1152,9 @@ type BypassPullRequestAllowances struct {
 type BypassPullRequestAllowancesRequest struct {
 	// The list of user logins allowed to bypass pull request requirements.
 	Users []string `json:"users"`
-	// The list of team slugs allowed to bypass pull request requirements.
+	// The list of teams allowed to bypass pull request requirements.
```

Comment:
Please revert this change.

![ksnip_20230114-072357](https://user-images.githubusercontent.com/6598971/212471466-422d1451-cb98-433a-a986-e552f5e1014c.png)